ui/components/src/LineChart/LineChart.tsx

```diff
@@ -170,10 +170,14 @@ export function LineChart({
         show: !isOptimizedMode,
         trigger: 'axis',
         showContent: false, // echarts tooltip content hidden since we use custom tooltip instead
-        axisPointer: {
-          type: isOptimizedMode ? 'none' : 'line',
-          z: 0, // ensure point symbol shows on top of dashed line
-        },
+      },
+      // https://echarts.apache.org/en/option.html#axisPointer
+      axisPointer: {
+        type: isOptimizedMode ? 'none' : 'line',
+        z: 0, // ensure point symbol shows on top of dashed line
+        triggerEmphasis: false, // https://github.com/apache/echarts/issues/18495
+        triggerTooltip: false,
+        snap: true,
       },
```

Comment:
I guess the `axisPointer` field moved to be less nested?

Reply:
IIRC, the nested one doesn't have as many options (e.g. the new `triggerEmphasis` I just added) and the top-level one is more comprehensive.